pkg/diagnostics/systemd/unit_status.go

```diff
@@ -24,13 +22,10 @@ func (d UnitStatus) Description() string {
 	return "Check status for related systemd units"
 }
 func (d UnitStatus) CanRun() (bool, error) {
-	if runtime.GOOS == "linux" {
-		if _, err := exec.LookPath("systemctl"); err == nil {
-			return true, nil
-		}
+	if IsSystemd() {
+		return true, nil
 	}
-
-	return false, errors.New("systemd is not present on this host")
+	return false, errors.New("systemd is not present/functional on this host")
```

Comment:
@sosiouxme is `false, error` more correct than `false, nil`?  I just don't recall.  There wasn't an error determining the status, we correctly determined it can't run